tests/Platforms/SqlitePlatformTest.php

```diff
@@ -360,6 +360,30 @@ public function testRenameNonExistingColumn(): void
         $this->platform->getAlterTableSQL($tableDiff);
     }
 
+    public function testRenameCamelCaseColumn(): void
```

Comment:
@eschricker thank you for the patch. Instead of testing SQL, please test the behavior at the integration level. See `RenameColumnTest`.

Reply:
I have updated the RenameColumnTest to include in the dataprovider the desired name for renaming in addition to the column name. But due to the comment that it's not working on case-sensitive platforms, the extended tests should fail on them.